docs/european-how-it-works.md

```diff
@@ -0,0 +1,19 @@
+---
+id: european-how-it-works
+title: European Protocol V1 How it Works 
+sidebar_label: How It Works
+slug: /european-v1/how-it-works
+---
+Unlike American-style options, European options expire at a set time, and cannot be exercised before that time. In exchange for this limitation, our European platform enables two important improvements over its American counterpart:
+
+#### Capital Efficiency 
+The European platform allows traders to write options without having to put the entire underlying asset up as collateral. For example, you might write a call for 1 BTC, but only need to deposit .15 BTC initially. This enables traders to make the most of their capital, particularly in allowing traders to hedge or speculate without tying up all of their assets. Efficiency doesn’t come without risk: if a position moves against the trader’s expectation, and they are unable to put up the required collateral to maintain it, that position may be liquidated. Keep an eye on your positions, and make sure your portfolio is able to meet its collateral requirements!
+
+#### Automatic Expiration
+Our American protocol allows traders to exercise at any time, but this also requires traders to manually exercise their option. European options expire automatically. Once the expiration date expires, traders can settle their position at the expired price, there is no need to remember to exercise if a trader intends to hold the option until expiration.
```

Comment:
probably best to not mention the American protocol so the reader doesn't need to understand anything about the Americans to understand the Euros